kong-ngx-build

```diff
@@ -289,7 +289,7 @@ main() {
             "--openssldir=$OPENSSL_INSTALL"
           )
 
-          if [[ $OPENSSL_VER == 1.1.* ]]; then
+          if version_eq $OPENSSL_VER 1.1; then
```

Comment:
* These `1.1.*` checks are a bit confusing. I thought it was being checked for exactly openssl versions `1.1.x`, and not `1.1.x` and beyond (`1.2`, `2.0`). Is that the case?
* Any comments on the version_eq check ignoring revisions if no revision_b has been provided? I thought it was useful to check things like `1.1.3` is `1.1.*`ish, but if it's not used, I would prefer to revert it to the straightforward equality.
* 👍 on `lte` and `gte`

Reply:
* This is just because OpenSSL 1.1.0 renamed the flag. It is most likely the flag will stick to this name going forward. 
* I don’t have any strong feelings. That’s fine imho. Although here we can say `>= 1.1.0`